src/android/libs/mobile_sdk/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Comment line immediately above ownership line is reserved for related gus information. Please be careful while editing.
```

Comment:
`mobile_sdk` houses the entirety of MSDK now rather than select modules.  Note, I did setup `update.sh` to exclude the git repository data.